torchmetrics/audio/stoi.py

```diff
@@ -131,3 +133,32 @@ def update(self, preds: Tensor, target: Tensor) -> None:  # type: ignore
     def compute(self) -> Tensor:
         """Computes average STOI."""
         return self.sum_stoi / self.total
+
+
+class STOI(ShortTermObjectiveIntelligibility):
```

Comment:
are there any deprecation tests?

Reply:
not for raising warning, just for the outputs, see https://github.com/PyTorchLightning/metrics/pull/752#discussion_r784751844